packages/compass-workspaces/src/provider.tsx

```diff
@@ -145,9 +149,9 @@ export const WorkspacesServiceProvider: React.FunctionComponent<{
           openWorkspaceAction({ type: 'My Queries' }, tabOptions)
         );
       },
-      openDatabasesWorkspace: (tabOptions) => {
+      openDatabasesWorkspace: (namespace, tabOptions) => {
         return store.dispatch(
-          openWorkspaceAction({ type: 'Databases' }, tabOptions)
+          openWorkspaceAction({ type: 'Databases', namespace }, tabOptions)
```

Comment:
I'm not sure how this makes sense. Namespace can't be applied to "Databases" workspace, this is an instance connection level workspace, not db / coll level

Reply:
I thought it would make sense for namespaces in the future to include the connection level as well. This is of course not implemented here, I only added this bit.
'connection1' is an item in the sidebar and will be highlighted when the user is on the list of 'connection1'. and 'db1.collection1' will be insufficient for multiconnection sidebar to correctly recognize which namespace to highlight, so the most straightforward way to me seemed to change namespaces so that they start from the connection level. I might be missing some context on what a namespace means outside of the sidebar.

Reply:
`namespace` has a defined meaning, it's specifically <db>.<coll> for Compass and generally when people talk about namespaces in MongoDB ([see glossary](https://www.mongodb.com/docs/manual/reference/glossary/)), we shouldn't mix connection id into the namespace

Reply:
If you want to start adding connection ids to the tab metadata in this PR, feel free to, but this should be a separate property on the tab options and added to all of them

Reply:
Oh I didn't know that, thank you! 

@kmruiz We were saying for the sidebar store that we could just extend the ids to include connection. We could still do it, but we'll also need to distinguish `namespace` and `connectionId` and send both in the actions:
```typescript
onNamespaceAction(
  evt.currentTarget.dataset.id as string,
  'select-connection'
);
```

Reply:
I removed the connection from namespace and left some TODOs for COMPASS-7775 to add activeConnection, but it can be also a separate task, or part of reworking the workspaces - what do you think @kmruiz ?

Reply:
In the sidebar list items it can be whatever you need it to be, this is just catching a DOM event with an id set on an html node and sending it up to be handled. But something inside the sidebar code should parse it to parts and provide further as two distinct values: connectionId and namespace. This is an implementation detail that should not affect the whole application

Reply:
Absolutely, as I said the `id` can still be combined. The part that needs to change is that `id` is not sent as `namespace`, it needs to be clear that the two are not interchangeable. Since we need to support both versions for a while, I'd separate them from the beginning to avoid confusion. Either having the `namespace` and `connectionId` as their own parameters, or use a different separator - e.g. `id: connection_db.coll`

Reply:
Currently the ConnectionInfo in single connection is global and can be acquired by using `useConnectionInfo`, and in multiple connections, we have the list of connections as parameters of the hook. I think we could adapt the store to always use a connectionId and relatively easily use the connectionInfo.id from whenever it comes as a separate parameter, as in the patch I shared, and we might not need to change the namespace. After taking a look, I'm leaning more to have a separate parameter for the connectionId, not changing the namespace (because we would overload it semantically, it would be either `db.coll` or `conn.db.coll`) and making the connectionId mandatory in the store in the form of `Record<ConnectionInfo['id'], whatever>`.

Reply:
Yes, changing the namespace apparently isn't an option. I have already reverted the changes. We could still have combined ids -> `db.coll` (collection item id in single connection usage) and `conn.db` (db item id in multi connection usage). Alternatively we could rename the ids to namespace, as they're not really identifiers for the menu item if they only contain `db.coll` level.